contract/AElf.Contracts.Consensus.AEDPoS/Types/MinerList.cs

```diff
@@ -32,7 +33,7 @@ public partial class MinerList
                 minerInRound.PublicKey = sortedMiners[i];
                 minerInRound.Order = i + 1;
                 minerInRound.ExpectedMiningTime =
```

Comment:
do not need change TimeStamp to DateTime, directly use TimeStamp + duration

Reply:
Should I add extensions for Timestamp? Like AddMilliseconds, AddSeconds then use duration to add inside the extensions?